packages/moveable/src/test/areEventsDragging.ts

```diff
@@ -31,7 +31,9 @@ describe('areEventsDragging', () => {
       clientY: 1,
       timeStamp: window.performance.now() + 10,
     };
-    expect(areEventsDragging(firstEvent, secondEvent)).toBeFalse();
+    expect(
+      areEventsDragging(firstEvent as MouseEvent, secondEvent as MouseEvent)
```

Comment:
This is a bit dirty, forcing the clearly _not-a-mouse-event_ to be one. Maybe we should create the events correctly through `new MouseEvent`.

I know that doesn't allow us to specify the timestamp, so we would have to use `jest.useFakeTimers()` and `jest.advanceTimersByTime()`. I actually don't know if it'll correctly advance the time inside the generated events, but I think it will?

This is definitely a nit though!

Reply:
When using fake timers, using `jest.advanceTimersByTime()` between two `new MouseEvent()` calls has no effect. At least in my testing. Does it work for you?

Reply:
I didn't actually try it out. But if it doesn't work, it's pretty useless then.